### DISCUSSION TITLE
Updating new changes to the dot files

### DISCUSSION THREAD

**Discussion by chiroro-jr** [end-4/dots-hyprland#179] [Q&A]
I am using the dot files in this repository for my Arch Linux desktop. I was wondering how I get updates in case the author changes some stuff in his dotfiles. Suppose maybe they changed the layout and behavior of the notification tray in a git commit, how do I update my system to show these changes?

**Comment by end-4** [comment]
You can run `install.sh` again
Thanks to the recent contributions of @clsty the script is idempotent. You can choose what to reinstall and what to not. It shows commands before execution and is very user-friendly.
Specifically for the behavior of notification tray, you wanna say yes when the script asks to sync `.config/ags`

**Reply by chiroro-jr** [reply]
Thank you for the feedback.

**Comment by clsty** [comment]
This is now documented [here](https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup/#updating):

> Automatically installed
>
> Run git pull under the git repo to get the latest update.
>
> Run ./install.sh again, and skip the steps which you don’t want (especially the rsync ... ones because it will override your files under its target path).
>
> To only update AGS, simply run ./update-ags.sh.
>
> Manually installed
>
> Run git pull under the git repo to get the latest update.
>
> Then grab things manually, similar to manual installation.
>
> To only update AGS, simply run ./update-ags.sh.

Closing this discussion in favor of the doc.
